flex-ui-core/src/main/java/org/cdlflex/ui/markup/html/repeater/data/table/TableBuilder.java

```diff
@@ -90,6 +90,17 @@ public TableBuilder() {
     }
 
     /**
+     * Sets a list of columns.
+     *
+     * @param columns the list of columns to add
+     * @return this for chaining
+     */
+    public TableBuilder<T, S> columns(List<IColumn<T, S>> columns) {
+        setColumns(columns);
```

Comment:
this essentially duplicates the behavior of setColumns, i think the method should behave like `column`, it should /add/ the columns to the builder. like using `addAll`.